src/components/dialog-full-screen/dialog-full-screen.component.js

```diff
@@ -112,6 +116,8 @@ class DialogFullScreen extends Modal {
           data-element='close'
           onClick={ this.props.onCancel }
           type='close'
+          tabIndex='0'
+          onKeyDown={ ev => (Events.isEnterKey(ev) ? this.props.onCancel() : null) }
```

Comment:
This should have `role=button` and should listen for `Enter` and `Space`. All close icons should be the same.

Reply:
Fixed